index.js

```diff
@@ -308,13 +308,11 @@ Blockchain.prototype.getBlock = function (blockTag, cb) {
   // determine BlockTag type
   if (Buffer.isBuffer(blockTag)) {
     lookupByHash(blockTag, cb)
-    return
   } else if (Number.isInteger(blockTag)) {
     async.waterfall([
       (cb) => lookupNumberToHash(blockTag, cb),
       (blockHash, cb) => lookupByHash(blockHash, cb)
     ], cb)
-    return
   } else {
     return cb(new Error('Unknown blockTag type'))
```

Comment:
If you remove `return` above, you can also remove it from here.

Reply:
? It's both removed. Or do you mean the ``return cb()``statement?

Reply:
Yep the return from this statement.

Reply:
So remove the whole ``else`` branch? Sorry, can't see atm why this is safe to do, is it clear that this is never reached?

Reply:
I mean that if you remove the `return` statement you can omit it from all three branches (but keep the `cb`).